File: Leetcode/2264 - Largest 3-Same-Digit Number in String/solution.js
var largestGoodInteger = function(num) {
    let res = "";
    for (let i = 2; i < num.length; i++) {
        if (num[i - 2] === num[i - 1] && num[i - 1] === num[i]) {
            const tempStr = num[i].repeat(3);
            if (tempStr > res) res = tempStr;
        }
    }
    return res;
};

File: Leetcode/2264 - Largest 3-Same-Digit Number in String/solution.cpp
string largestGoodInteger(string num) {
    string res = "";
    for (int i = 2; i < num.size(); i++) {
        if (num[i-2] == num[i-1] && num[i-1] == num[i]) {
            string tempStr(3, num[i]);
            if (tempStr > res) res = tempStr;
        }
    }
    return res;
}